ci/utility/jobs/dev-object-tagging.yml

```diff
@@ -0,0 +1,36 @@
+jobs:
+  - name: rbac-csv-upload-dev
+    max_in_flight: 1
+    serial: true
+    plan:
+      - get: rbac-csv
+        trigger: true
+      - get: aws-common-infrastructure
+      - .: (( inject meta.plan.terraform-output ))
+      - .: (( inject meta.plan.rbac-csv-upload ))
+        config:
+          params:
+            AWS_ROLE_ARN: arn:aws:iam::((aws_account.development)):role/ci
+
+  - name: object-tagging-dev
+    max_in_flight: 1
+    serial: true
+    plan:
+      - get: rbac-csv
+        trigger: true
+        passed:
+          - rbac-csv-upload-dev
+      - get: dataworks-s3-object-tagger
+        trigger: true
+      - put: meta
+        resource: meta
```

Comment:
you should have separate meta resources per environment, otherwise they clash if running at the same time

Reply:
Didn't know that